### PR TITLE
80 sidepanel opening post upload

### PR DESCRIPTION
**Pull request by nasouh94**
### Overview
when uploading new media we dont open the sidepanel.

[staging](https://tce.7.plankdesign.com/)

### Work was done
1. i added the logic to the trigger the side panel when a new upload is made. NOTE we only show the sidepanel when one image has been uploaded, if the user uploads multiple ones it'll remain closed.
2. i had to add new logic to take into account the user updating values of the new uploaded media. so i added an isNewMedia value to differentiate between new and just updated media 

### to test
1. login to the CMS on [staging](https://tce.7.plankdesign.com/admin/media)
2. navigate to the media manager
3. upload a new image
4. the sidepanel should open up and you should be able to update the values of the new image with a success message showing up after you save

### CODE REVIEW

File: resources/js/components/modals/files/mm-modal-add.vue [hunk 1]
```diff
@@ -33,7 +33,7 @@
             <vue-dropzone
                 ref="myVueDropzone"
                 :style="styleBtnDefault"
-                v-on:vdropzone-success="uploadSuccess($event)"
+                v-on:vdropzone-success="uploadSuccess"
```
Comment (meganmcewan) [inline resources/js/components/modals/files/mm-modal-add.vue:36]: can the message read "saved" or just "success" instead of "uploaded" when you hit save?

Comment (meganmcewan) [reply resources/js/components/modals/files/mm-modal-add.vue:36]: Also it feels like the modal should close after you hit save. what do you think?

**Comment by m-triassi** [comment]
Oh i can answer some of those @meganmcewan 
> where does the image go once it's uploaded?

The images are uploaded either to the local disk, or to amazon S3 by the Media manager API that the front end interacts with

> how are the images ordered?

Images are ordered in a first in first out ordering, so basically they're ordered in the order you attach them, or for use in the WYSIWYG it's arbitrary :)

File: resources/js/components/modals/files/mm-modal-add.vue (hunk 1 above)
Comment (nasouh94) [reply resources/js/components/modals/files/mm-modal-add.vue:36]: it did cross my mind but they might want to input the french values too. 

right now how it follows this flow:

1. attach image => triggers sidepanel to open
2. add english value and save 
3. toggle to french values 
4. add them and save

Comment (meganmcewan) [reply resources/js/components/modals/files/mm-modal-add.vue:36]: makes sense!

i would just update the success message if possible then

**Review by meganmcewan — APPROVED** [review]

**Review by m-triassi — APPROVED** [review]
I think megan has covered all the bases, here. This looks excellent to me!

**Pull request merged**